src/reducers/sources.js

```diff
@@ -22,7 +21,8 @@ export type SourcesState = {
   selectedLocation?: {
     sourceId: string,
     line?: number,
-    column?: number
+    column?: number,
+    url: string
```

Comment:
hmm, lets avoid adding `url` here. I think we should be able to get the source where its being used